mobile/library/common/config/config.cc

```diff
@@ -496,28 +496,14 @@ stats_sinks: *stats_sinks
   stats_matcher:
     inclusion_list:
       patterns:
-        - safe_regex:
-            regex: '^cluster\.[\w]+?\.upstream_cx_[\w]+'
-        - safe_regex:
-            regex: '^cluster\.[\w]+?\.upstream_rq_[\w]+'
-        - safe_regex:
-            regex: '^cluster\.[\w]+?\.update_(attempt|success|failure)'
-        - safe_regex:
-            regex: '^cluster\.[\w]+?\.http2.keepalive_timeout'
-        - safe_regex:
-            regex: '^dns.apple.*'
```

Comment:
disable dns.apple - I added it but it has never been used by anybody I think so should be safe to remove it.